src/client/entity-editor/identifier-editor/identifier-row.tsx

```diff
@@ -185,8 +194,10 @@ function mapDispatchToProps(
 		onRemoveButtonClick: () => dispatch(removeIdentifierRow(index)),
 		onTypeChange: (value: {value: number}) =>
 			dispatch(updateIdentifierType(index, value && value.value)),
-		onValueChange: (event: React.ChangeEvent<HTMLInputElement>) =>
-			handleValueChange(dispatch, event, index, typeOptions)
+		onValueChange: (event: React.ChangeEvent<HTMLInputElement>) =>{
+			const { cleanedValue, action } = handleValueChange(event,index,typeOptions);
```

Comment:
I'm looking at this again, and I see we don't do anything with `cleanedValue`.
Are these changes to `handleValueChange` and `mapStateToProps` required at all?

Reply:
Oh yes, I see the cleanedValue is actually doing nothing. The reason I had used it earlier was that I was planning to pass it as a prop and then pass it further to the "IdentifierLink" but later realised that this is not required, things being defined the way they are, 'typevalue' is already modified to have the cleaned upvalue.

Reply:
Hi @insane-22, Do you have a minute to finish this PR off?
I think apart from the handleValueChange and mapStateToProps changes it should be ready to merge.

Reply:
yes yes @MonkeyDo , I sincerely apologise for this delay. It should have been done by now. I would be doing the changes now.